package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -148,6 +148,37 @@ local function fixup_wan(radio, index)
 	uci:set('wireless', name, 'macaddr', macaddr)
 end
 
+-- First we need to count all radios with a fixed frequency band
+-- This is necessary as devices which have radios capable of operating
+-- in the 2.4 GHz and 5 GHz band need to be distributed evenly
+local radio_band_count = {band24=0, band5=0}
+util.foreach_radio(uci, function(radio, index, config)
+	local hwmodes = iwinfo.nl80211.hwmodelist(util.find_phy(radio))
+	if hwmodes.a and hwmodes.g then
+		-- Dualband - do nothing in this step
+	elseif hwmodes.a then
+		-- 5 GHz
+		radio_band_count["band5"] = radio_band_count["band5"] + 1
+	elseif hwmodes.g then
+		radio_band_count["band24"] = radio_band_count["band24"] + 1
+	end
+end)
+
+-- Now we have the number of all fixed 2.4G GHz and 5 GHz radios.
```

Comment:
"2.4G GHz" ?
and i wouldnt write "now we have.." but rather something like "Using the number of all ..."